src/Files.Uwp/Views/LayoutModes/DetailsLayoutBrowser.xaml

```diff
@@ -217,7 +217,6 @@
                         VerticalContentAlignment="Stretch"
                         tui:ScrollViewerExtensions.EnableMiddleClickScrolling="{x:Bind IsMiddleClickToScrollEnabled, Mode=OneWay}"
                         x:FieldModifier="public"
-                        AllowDrop="{x:Bind InstanceViewModel.IsPageTypeSearchResults, Converter={StaticResource BoolNegationConverter}, Mode=OneWay}"
```

Comment:
Does this need to be removed?

Reply:
Will re-add, as that wasn't the cause.

Reply:
> Does this need to be removed?

I'll re-add it in a bit if it wasn't the cause of the issue.

Reply:
@yaichenbaum that seemed to bring the issue back after bringing it again, so I had to remove it.

Reply:
Because the default value for AllowDrop is false, this change prevents dragging items and dropping them into the blank space to move them to the current directory.

Reply:
> Because the default value for AllowDrop is false, this change prevents dragging items and dropping them into the blank space to move them to the current directory. Are you sure this is the cause of the issue?

Yes, removing it makes dragging work from what i can see.